qutebrowser/browser/browsertab.py

```diff
@@ -196,6 +196,10 @@ def next_result(self, *, result_cb=None):
         """
         raise NotImplementedError
 
+    def searching(self):
+        """Return True if we are currently searching or not."""
+        raise NotImplementedError
+
```

Comment:
I don't think it makes sense to have a getter method for this - just do `self.searching = False` in `AbstractSearch.__init__` and set that instead of `_searching` below.